CONTRIBUTING.md

```diff
@@ -1,62 +1,181 @@
-# Node.js Community Contributing Guide 1.0
+# Node.js Website Contributing Guide
 
-## Code of Conduct
+Thank you for your interest in contributing to the Node.js Website. Before you proceed, briefly go through the following:
 
-The Code of Conduct explains the _bare minimum_ behavior
-expectations the Node Foundation requires of its contributors.
-[Please read it before participating](https://github.com/nodejs/admin/blob/HEAD/CODE_OF_CONDUCT.md).
+* [Code of Conduct](https://github.com/nodejs/node/blob/HEAD/CODE_OF_CONDUCT.md)
+* [Getting started](#getting-started)
+  * [Vocabulary](#vocabulary)
+  * [Commit message guidelines](#commit-guidelines)
+  * [Pull Request Policy](#pull-request-policy)
+    * [Before merging](#before-merging)
+    * [When merging](#when-merging)
+  * [Becoming a collaborator](#becoming-a-collaborator)
+  * [Debugging failing checks](#debugging-failing-checks)
+  * [Developer's Certificate of Origin 1.1](#developers-certificate-of-origin-11)
+  * [Remarks](#remarks)
 
-## Vocabulary
+## Getting started
+
+1. Click the fork button in the top right to clone the [nodejs.org repository](https://github.com/nodejs/nodejs.org/fork)
+
+2. Clone your fork using SSH, GitHub CLI, or HTTPS.
+
+  ```bash
+  git clone git@github.com:<GITHUB_ID>/nodejs.org.git # SSH
+  gh repo clone <GITHUB_ID>/nodejs.org # GitHub CLI
+  git clone https://github.com/<GITHUB_ID>/nodejs.org.git # HTTPS
+  ```
+
+3. Change into the nodejs.org directory.
+
+  ```bash
+  cd nodejs.org
+  ```
+
+4. Create a remote for keeping your fork as well as your local clone up-to-date.
+
+  ```bash
+  git remote add upstream git@github.com:nodejs/nodejs.org.git
+  ```
+
+5. Create a new branch for your work.
+
+  ```bash
+  git checkout -b name-of-your-branch
+  ```
+
+6. Run the following to install the dependencies and start a local preview of your work.
+
+  ```bash
+  npm install # installs this project's dependencies
+  npm start # starts a preview of your local changes
+  ```
+
+7. Perform a merge to sync your current branch with the upstream branch.
+
+ ```bash
+git fetch upstream
+git merge upstream/main
+```
+
+8. Run `npm test` to confirm that tests, linting, and formatting are passing. See [here](#-debugging-failing-checks) to fix failures. Ensure that your tests are passing before making a PR.
```

Comment:
We don't have a test script in this repo atm

Reply:
I think we can put it in comment and uncomment when we add unit test on this repo. What do you think?

Reply:
Comments aren't part of the CommonMark specification, I will remove them.